client.go

```diff
@@ -87,18 +79,8 @@ func New(base_url, token string) (*Client, error) {
 func newClient(url, grafana_url string) (*Client, error) {
 	c := &Client{}
 
-	// Configure the HTTP client.
-	c.client = &retryablehttp.Client{
-		Backoff:      c.retryHTTPBackoff,
-		CheckRetry:   c.retryHTTPCheck,
-		RetryWaitMin: 100 * time.Millisecond,
-		RetryWaitMax: 400 * time.Millisecond,
-		RetryMax:     5,
-	}
-	// https://grafana.com/docs/grafana-cloud/oncall/oncall-api-reference/#rate-limits
-	baseLimit := 50.0 / 60
-	limit := rate.Limit(baseLimit)
-	c.limiter = rate.NewLimiter(limit, 50)
+	// retryablehttp.Client will retry up to 4 times on recoverable errors (429, 5xx, and low-level network errors)
+	c.client = retryablehttp.NewClient()
```

Comment:
the default configuration is good enough. it will retry on 5xx and 429s, respecting the `Retry-After` header.